### DISCUSSION TITLE
why doesnt Cheerio work when I deployed project on Cyclic?

### DISCUSSION THREAD

**Discussion by SheikhElMoctarG** [cheeriojs/cheerio#3210] [Q&A]
Hi, I write a project using cheerio and nodejs, it works on my computer but it doesnt work on Cyclic, how to fix it and it is happening ?

**Comment by fb55** [comment]
Hi @SheikhElMoctarG, please provide some more context around what is happening. This is the first time I've heard of Cyclic, and this is not enough information for me to say anything.
